### PR TITLE
BaSP-TG-80: Admins get with Redux

### PR DESCRIPTION
**Pull request by MaicolFreire001**
**Admin react redux:**
- Create constants to get admins
- Create actions to get admins
- Create thunk to get admins
- Create admins reducer
- Connect list component with store to get admins

### CODE REVIEW

**Review by fede230298 — APPROVED** [review]

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -3,9 +3,13 @@ import styles from './admins.module.css';
 import MessageModal from '../Shared/Modal/MessageModal';
 import Table from '../Shared/Table';
 import Button from '../Shared/Button/Button';
+import { useSelector, useDispatch } from 'react-redux';
+import { getAdmins } from '../../redux/admins/thunks';
 
 const Admins = () => {
```
Comment (Ignacio-Sormani) [inline src/Components/Admins/index.js:9]: Remove comments if they are not necessary

Comment (MaicolFreire001) [reply src/Components/Admins/index.js:9]: Done

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -58,10 +52,18 @@ const Admins = () => {
     { heading: 'Actions' }
   ];
 
+  if (isLoading) {
+    return <h2>Loading...</h2>;
+  }
+
+  if (error) {
```
Comment (Ignacio-Sormani) [inline src/Components/Admins/index.js:59]: Maybe use the shared info modal to display the error

Comment (MaicolFreire001) [reply src/Components/Admins/index.js]: Done

**Review by JuliMonte5 — APPROVED** [review]

File: src/redux/rootReducer.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import { combineReducers } from 'redux';
-
+import AdminReducer from './admins/reducer';
 import timeSheetsReducer from './timeSheets/reducer';
 import projectsReducer from './projects/reducer';
 
 const rootReducer = combineReducers({
+  admins: AdminReducer,
```
Comment (ProDIGGY-vinilos) [inline src/redux/rootReducer.js:7]: The 'A' of AdminReduce should be lowercase

Comment (MaicolFreire001) [reply src/redux/rootReducer.js:7]: Done

**Pull request merged**